chore(travis): don't fail generating docs when there's no git version

zetok opened the pull request:
<!-- Reviewable:start -->

This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/3836)

<!-- Reviewable:end -->

zetok committed:
chore(travis): don't fail generating docs when there's no git version

Can happen due to travis cloning only 50 latest revisions.

Diadlo commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[.travis/build-docs.sh, line 26 at r1](https://reviewable.io:443/reviews/qtox/qtox/3836#-KV24w6YxwLpVjkr3EIi:-KV24w6Z9je8v3EhXXbZ:b-65kegb) ([raw file](https://github.com/qtox/qtox/blob/8a72928ad1451a8a76151a1a070fb35c022aa53b/.travis/build-docs.sh#L26)):_

> ``` Shell
> 
> # can fail due to travis cloning only `depth=50`
> GIT_DESC=$(git describe --tags 2>/dev/null || echo HEAD)
> ```

Why did you remove space between `2>` and `/dev/null`?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3836)_

<!-- Sent from Reviewable.io -->

zetok commented:
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[.travis/build-docs.sh, line 26 at r1](https://reviewable.io:443/reviews/qtox/qtox/3836#-KV24w6YxwLpVjkr3EIi:-KV263Fd5uTKlDt5nNs0:b24f5hk) ([raw file](https://github.com/qtox/qtox/blob/8a72928ad1451a8a76151a1a070fb35c022aa53b/.travis/build-docs.sh#L26)):_
<details><summary>

<i>Previously, Diadlo (Polshakov Dmitry) wrote…</i></summary>



> Why did you remove space between `2>` and `/dev/null`?
> </details>
> I've removed it all while checking why things fail, and while re-adding I just added it without the space.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3836)_

<!-- Sent from Reviewable.io -->

Diadlo commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3836#-:-KV2CCDZFcwcyMSYiJNy:bnfp4nl)_

<!-- Sent from Reviewable.io -->
